LiskHQ/lisk-sdk
# Un-skip verify integration tests - Closes #3473

### What was the problem?

framework/test/mocha/integration/blocks/verify/verify.js contained skipped tests due to new transactions implementation.

### How did I fix it?

- by un-skipping and fixing the tests

### How to test it?

run `npm run mocha:integration test/mocha/integration/blocks/verify/verify.js` and all tests but the verifyId block should pass

### Review checklist

* The PR resolves #3473
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
